app/components/agenda/agenda-list/list-item/template.hbs

```diff
@@ -38,8 +45,8 @@
         {{#if (await agendaitem.checkAdded)}}
           <i class="vl-link__icon vl-vi vl-vi-calendar-add"></i>
           {{#attach-popover
-             placement="top"
-             class="ember-attacher-popper ember-attacher-popper-wide"
+                  placement="top"
+                  class="ember-attacher-popper ember-attacher-popper-wide"
```

Comment:
Why was the indentation changed? I feel as though the editor might have done this for you. We don't really have a written convention around this. But I thought we just indented 2 spaces extra every time instead of alignment like what happened here?

Reply:
Automatic code formatting in de editor.

Reply:
It's true though that this is somewhat inconsistent with the style of rest of the codebase (and what you see most in the wild for htmlbars). Changing formatter config for next time would be nice? :-)